tests/t5/test_modeling_tf_t5.py

```diff
@@ -496,7 +480,10 @@ def test_greedy_xla_generate_simple(self):
         output_strings = tokenizer.batch_decode(output_ids, skip_special_tokens=True)
         output_strings_xla = tokenizer.batch_decode(output_ids_xla, skip_special_tokens=True)
 
-        expected_output_string = ["Heute ist ein schöner Tag."]
+        expected_output_string = [
+            "Heute ist ein schöner Tag.",
+            "Ich habe vier Katzen, drei Hunde, zwei Vögel und ein Pferd.",
+        ]
```

Comment:
Are we guaranteed that the logits are stable enough that we'll always sample this exact output? A flaky test can be really annoying!

Reply:
We haven't had problems with similar tests, so I'm assuming we won't have problems :D

Reply:
...fair point!